src/groupIssues.ts

```diff
@@ -48,7 +48,7 @@ const handleIssueGroups = (allIssuesCreated: QueryGroup[], allIssueComments: Que
             // make sure that comment belongs to a PR group
             if (!finalIssues.secondary[issueUrl]) {
                 finalIssues.secondary[issueUrl] = {
-                    groupTitle: `Left comments on issue: [${comment.issue.title}](${issueUrl}) in ${repoGroup.repo}`,
+                    groupTitle: `Left comments on issue: [${comment.issue.title}](https://github.com${issueUrl}) in ${repoGroup.repo}`,
```

Comment:
This looks ok 👍 We just need some updated tests please :)

Reply:
Added a test for this